### ISSUE TITLE
Questions about normalization

### ISSUE THREAD

**Issue by garyfanhku** [implus/PytorchInsight#2]
Hi, 

Massive thanks for the enlightening research and the repo! Just wondering what's the reason behind using a self-implemented BatchNorm2d [here](https://github.com/implus/PytorchInsight/blob/bf76edd9f496858686c60fbf6d8a354cc72f913b/classification/models/imagenet/resnet_sge.py#L21)?

**Comment by implus** [comment]
Hi, that's actually not a BatchNorm2d here (in fact a GroupNorm) since we donot calculate moving means and variances. You can also use GroupNorm instead.

**Comment by garyfanhku** [comment]
Thanks a lot for clarifying!
